build_dockers.bsh

```diff
@@ -20,26 +20,26 @@ export GOLANG_VERSION
 export DOCKER_LFS_BUILD_VERSION=${DOCKER_LFS_BUILD_VERSION:-release-1.3}
 
 if [[ $# == 0 ]]; then
-  IMAGE_NAMES=($(ls -d ${CUR_DIR}/*/*/Dockerfile))
+  IMAGE_NAMES=($(ls -d ${CUR_DIR}/*.Dockerfile))
 else
   IMAGE_NAMES=("${@}")
 fi
 
 #This will take a long time the first time
 for IMAGE_NAME in "${IMAGE_NAMES[@]}"; do
-  # IMAGE_NAME is "./debian/7/Dockerfile"
+  # IMAGE_NAME is "./debian_7.Dockerfile"
   # strip basename
-  # strip leading non whitespace
-  # convert / to _
+  # strip leading "./"
+  # strip trailing ".Dockerfile"
   IMAGE_DIR=$(dirname ${IMAGE_NAME})
-  NAME=$(printf $IMAGE_DIR | sed 's/[\.\/]*//' | sed 's/\//_/')
+  NAME="$(printf $IMAGE_NAME | sed 's/\.\///' | sed 's/\.Dockerfile$//')"
```

Comment:
Thanks. I'm going to leave this as-is for now, since I can more clearly see what's going on with the two `sed`s.